static/css/components/carousel.less

```
@@ -26,6 +26,13 @@
   overflow-x: scroll;
   padding: 10px 20px;
 
+  &.carousel--grid {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(150px, 1fr));
+    grid-gap: 30px;
+    // Fallback for non-grid browsers
+    flex-wrap: wrap;
+  }
```

And some small tweaks to make this consistent with our other carousels, which currently display 6 books on desktop, 2 books on mobile.

```suggestion
  &.carousel--grid {
    display: grid;
    grid-template-columns: repeat(auto-fill, minmax(135px, 1fr));
    grid-gap: 30px 10px;
    // Fallback for non-grid browsers
    flex-wrap: wrap;

    .carousel__item {
        margin: 0; // Handled by grid gap
    }
  }
```